cluster/swarm/cluster.go

```diff
@@ -241,6 +241,19 @@ func (c *Cluster) RemoveNetwork(network *cluster.Network) error {
 		for _, engine := range c.engines {
 			engine.DeleteNetwork(network)
 		}
+	} else if strings.Contains(err.Error(), "Is the docker daemon running") {
+		log.Debug("Some of the original engines unreachable - Rmoving global network from the current engines...")
+		for _, engine := range c.engines {
+			e1 := engine.RemoveNetwork(network)
+			if e1 == nil {
+				for _, engine := range c.engines {
+					engine.DeleteNetwork(network)
+				}
+				err = nil
+				break
+			}
```

Comment:
I mean if engine.RemoveNetwork(network) returned a non-nil err, program did nothing. Maybe it is not good.